doc/api/tls.md

```diff
@@ -1233,6 +1267,20 @@ changes:
   certificate from a connecting client. Only applies when `isServer` is `true`.
 * `rejectUnauthorized` {boolean} If not `false` a server automatically reject clients
    with invalid certificates. Only applies when `isServer` is `true`.
+* `pskCallback(hint)` {Function} When negotiating TLS-PSK, this optional
+   function is called with the identity hint provided by the server. If the
+   client should continue to negotiate PSK ciphers, the return value of this
+   function must be an object in the form `{psk: <string>, identity:
+   <string>}`.
+   * *Note*: PSK ciphers are disabled by default, and using
+   TLS-PSK thus requires explicitly specifying a cipher suite with the
+   `ciphers` option. Additionally, it may be necessary to disable
+   `rejectUnauthorized` if a client attempts to specify a certificate for the
+   session.
+   * *Note*: `psk` must be a hexadecimal string.
+* `pskIdentity`: {string} The identity hint to send to clients when
+  negotiating TLS-PSK.
```

Comment:
Non-blocking suggestion: I would prefer to call this `pskIdentityHint`. Depending on the application, identity hints may use a different format than identities, and even if they use the same format, their values may differ.